lib/active_admin/application.rb

```diff
@@ -271,10 +271,12 @@ def controllers_for_filters
   private
 
     def register_default_assets
-      register_stylesheet 'active_admin.css',       media: 'screen'
-      register_stylesheet 'active_admin/print.css', media: 'print'
+      ActiveSupport::Deprecation.silence do
```

Comment:
Haha now you tell me! After I make the change! 🙄😂 Its done and if anything better change for when we remove this as we can just either remove asset registration or just use the module to have the javascripts and stylesheets methods.